data/json/professions.json

```
@@ -767,6 +767,29 @@
       "female": [ "bra", "panties" ]
     }
   },
+  {
+    "type": "profession",
+    "id": "butcher",
+    "name": "Butcher",
+    "description": "You spent most of your adult life in a butcher shop. Your trusty knife has seen many different creatures and you know how to dissect them. ",
```

```suggestion
    "description": "You spent most of your adult life in a butcher shop.  Your trusty knife has seen many different creatures and you know how to butcher them.",
```

Double space + wording. Yeah, the wording on the last part is nit-picky, but I never heard any butcher say they "dissected" something. Then again, I've not talked to many butchers that spoke English...